libmscore/layout.cpp

```diff
@@ -2682,9 +2681,18 @@ void Score::connectTies(bool silent)
                                           spanner->setParent(initialNote);
                                           initialNote->add(spanner);
                                           }
-                                    else
+                                    else {
                                           delete spanner;
+                                          }
                                     }
+                              }
+                        // glissando with no end element can happen during copy/paste
+                        for (Spanner* spanner : n->spannerFor()) {
+                              if (spanner->type() == Element::Type::GLISSANDO && spanner->endElement() == nullptr) {
```

Comment:
It's probably worth removing any spanner with no end element here. Glissando or not